Path/IO.hs

```diff
@@ -1,3 +1,4 @@
+-- vim:ts=2 sw=2
```

Comment:
Please remove this. I prefer not to keep editor-specific settings in source files.